src/store/media.js

```diff
@@ -247,14 +259,17 @@ export const getters = {
     if (getters.searchType === ALL_MEDIA) {
       return {
         isFetching:
-          state.fetchState[AUDIO].isFetching ||
-          state.fetchState[IMAGE].isFetching,
+          supportedMediaTypes.filter(
+            (type) => state.fetchState[type].isFetching
+          ).length > 0,
         fetchError:
-          state.fetchState[AUDIO].fetchError ||
-          state.fetchState[IMAGE].fetchError,
+          supportedMediaTypes.filter(
+            (type) => !!state.fetchState[type].fetchingError
+          ).length > 0,
         isFinished:
-          state.fetchState[AUDIO].isFinished &&
-          state.fetchState[IMAGE].isFinished,
+          supportedMediaTypes.filter(
+            (type) => state.fetchState[type].isFinished
+          ).length === supportedMediaTypes.length,
```

Comment:
I realized later that for `fetchError` this is actually incorrect: when we have an error for one type, the return will be true, instead of the error message.